Replace mention of darksky as replacement

jagee23 opened the pull request:
I think for users looking through the docs it would be more helpful to suggest NWS as a replacement rather than darksky since it is no longer a viable option for new users.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

jagee23 committed:
Replace mention of darksky as replacement

I think for users looking through the docs it would be more helpful to suggest NWS as a replacement rather than darksky since it is no longer a viable option for new users.

fabaff commented on source/_integrations/yweather.markdown:
```diff
@@ -11,7 +11,7 @@ ha_domain: yweather
 <div class='note warning'>
 
-The Yahoo Weather API is being [retired](https://developer.yahoo.com/weather/?guccounter=1). A replacement is the [`darksky` sensor](/integrations/darksky).
+The Yahoo Weather API is being [retired](https://developer.yahoo.com/weather/?guccounter=1). A replacement is the [`NWS` integration](/integrations/nws).
```
I suggest to choose a replacement which is not limited to the US.

jagee23 replied:
Yes, that is a very good point.  I believe OpenWeatherMap would be a better suggestion.

jagee23 committed:
Replace suggested alternative 

Replace NWS suggestion with OpenWeatherMap.  OpenWeatherMap includes data for countries other than US.

frenck reviewed (approved):
Thanks, @jagee23 👍
